### ISSUE TITLE
Add ariaDescribedBy property to config

### ISSUE THREAD

**Issue by Confidenceman02** [Confidenceman02/elm-select#17]
# Context
Currently we do not set a ariaDescribedBy property on the select input and should be doing so for accessibility compliance.

# Work to be done
- Add ability to set an ariaDescribedBy property on all select variants.
